recipes/interpolation/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.8" %}
+{% set sha256 = "2debe8a5fc000668edbf1b16bdc8de6e8830047bc3f3ffcb2a5636c16137b004" %}
+package:
+  name: interpolation
+  version: {{ version }}
+source:
```

Comment:
Please add a line between the main section on the recipe. See the example recipe.